website/docs/r/vra_load_balancer.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 # Resource: vra_load_balancer
 
 Creates a VMware vRealize Automation load balancer resource.
-
+x
```

Comment:
Remove

Reply:
Resolved in [79cbb38](https://github.com/vmware/terraform-provider-vra/pull/575/commits/79cbb38e0198b4deb252dc026c8d3189b981e5d0).